src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -2126,13 +2186,6 @@ private List<CompletionResult> GetResultForIdentifier(CompletionContext completi
             replacementIndex = completionContext.ReplacementIndex;
             replacementLength = completionContext.ReplacementLength;
 
-            if (result.Count == 0
-                && completionContext.TokenAtCursor.TokenFlags.HasFlag(TokenFlags.TypeName)
-                && lastAst?.Find(a => a is MemberExpressionAst, searchNestedScriptBlocks: false) is not null)
```

Comment:
why is this removed? Is this scenario already covered somewhere else?

Reply:
Yes, it's handled like this:
```
if (typeNameToComplete is null && tokenAtCursor?.TokenFlags == TokenFlags.TypeName)
{
    typeNameToComplete = new TypeName(tokenAtCursor.Extent, tokenAtCursor.Text);
}
```
on line 963 (currently).